src/main/java/reactor/netty/channel/MicrometerChannelMetricsRecorder.java

```diff
@@ -60,7 +60,7 @@
 	final ConcurrentMap<MeterKey, Timer> addressResolverTimeCache = PlatformDependent.newConcurrentHashMap();
 
 
-	public MicrometerChannelMetricsRecorder(String name, String protocol) {
+	protected MicrometerChannelMetricsRecorder(String name, String protocol) {
```

Comment:
this is an API breaking change, since the class itself is public. but we already made a similar breaking change in 0.9.3 (the reverse actually, removing a `protected` 3 args constructor to add the current `public` constructor)...

Reply:
ok I'll revert it to `public`